### ISSUE TITLE
about loss_vq

### ISSUE THREAD

**Issue by PonyPinkPie** [xcyao00/ResAD#11]
hi, why we have to define loss_vq like this `loss = torch.mean((z_q.detach() - z_normal) ** 2) + self.beta * torch.mean((z_q - z_normal.detach()) ** 2)`,. is there any explanation for this part, thank you :)
